src/networks.js

```diff
@@ -231,5 +231,27 @@ module.exports = {
       4: 0x76b809bb
     },
     coin: coins.KMD
+  },
+  decred: {
+    messagePrefix: 'Decred Signed Message:\n',
+    bip32: {
+      public: 0x02fda926,
+      private: 0x02fda4e8
+    },
+    pubKeyHash: 0x073f,
+    scriptHash: 0x071a,
+    wif: 0,
```

Comment:
correct?

Reply:
AFAICT this is incorrect.

`wif` here seems to be the "wif version" used by bitcoinjs/wif (this is used [in ecpair](https://github.com/trezor/trezor-utxo-lib/blob/trezor/src/ecpair.js#L95) referring to [bitcoinjs/wif](https://github.com/bitcoinjs/wif/blob/78ad28eecb96f5a08a76430b24460ad48706b904/index.js#L11)).

Decred doesn't use "version" but it does have the network id sort of performing the same role. It also has a "scheme" field to specify the specific crypto algo used with that key (secp256k1, ed25519, etc) (https://github.com/decred/dcrd/blob/b60c60ffe98bcea5becc4e9e4d2a4fe6152401f6/dcrutil/wif.go#L148)

Reply:
thanks, found it